src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/CreateClaimCallbackHandler.java

```diff
@@ -87,18 +90,20 @@
 
     @Override
     protected Map<String, Callback> callbacks() {
-        return Map.of(
-            callbackKey(ABOUT_TO_START), this::emptyCallbackResponse,
-            callbackKey(MID, "eligibilityCheck"), this::eligibilityCheck,
-            callbackKey(MID, "applicant"), this::validateDateOfBirth,
-            callbackKey(MID, "fee"), this::calculateFee,
-            callbackKey(MID, "idam-email"), this::getIdamEmail,
-            callbackKey(MID, "particulars-of-claim"), this::validateParticularsOfClaim,
-            callbackKey(MID, "appOrgPolicy"), this::validateApplicantSolicitorOrgPolicy,
-            callbackKey(MID, "repOrgPolicy"), this::validateRespondentSolicitorOrgPolicy,
-            callbackKey(ABOUT_TO_SUBMIT), this::submitClaim,
-            callbackKey(SUBMITTED), this::buildConfirmation
-        );
+        HashMap<String, Callback> callbackMap = new HashMap<>();
```

Comment:
Just curious about, replacement of `Map.of(..)` with mutable map here.

Reply:
Map.of method can only take 10 items, I will change it back when I remove the other callback 😆